diff --git a/tests/tests/test_state_scripts.py b/tests/tests/test_state_scripts.py
index 7db7351..6888cc0 100644
--- a/tests/tests/test_state_scripts.py
+++ b/tests/tests/test_state_scripts.py
@@ -630,6 +630,7 @@ class TestStateScripts(MenderTesting):
     ]
 
     @MenderTesting.slow
+    @pytest.mark.skip(reason="WHY?????")
     @pytest.mark.usefixtures("standard_setup_one_client_bootstrapped")
     @pytest.mark.parametrize("test_set", TEST_SETS)
     def test_state_scripts(self, test_set):
